app/assets/javascripts/discourse/tests/acceptance/tags-test.js

```diff
@@ -19,6 +87,13 @@ acceptance("Tags", function (needs) {
       "shows the eviltrout tag"
     );
   });
+
+  test("dismiss notifications", async function (assert) {
+    await visit("/tag/test/l/unread");
+    await click("button.dismiss-read");
+    await click(".dismiss-read-modal button.btn-primary");
+    assert.ok(find(".dismiss-read-modal:visible").length === 0);
```

Comment:
```suggestion
    assert.ok(query(".dismiss-read-modal:visible"));
```

(you'll need to add the import, just before `queryAll` one)

Reply:
What if I used `visible`?

Reply:
Ah, even better 😅